app/experimenter/outcomes/README.md

```diff
@@ -0,0 +1,36 @@
+# Outcomes
+
+## Overview
+
+[Outcomes](https://github.com/mozilla/jetstream-config/#outcome-snippets) define automated analyses that can be applied to Nimbus experiments. They are stored in the [Jetstream Config repo](https://github.com/mozilla/jetstream-config/tree/main/outcomes) as [TOML](https://github.com/toml-lang/toml) files and are consumed by Experimenter as a [Git Submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
+
+## Building and Deploying
+
+Every time the Experimenter Docker container is built, it will synchronize the Jetstream Config repo into the local Experimenter source tree, which will then be copied with the rest of Experimenter's source into the Docker container. The Outcomes module will load the outcome definition TOMLs into memory where they can be accessed.
+
+## Using Outcomes
+
+Outcomes can be accessed by using the Outcome and Outcomes classes. An Outcome has the following properties:
+
+- `app_id: str` An application ID defined as one of `("fenix", "firefox-desktop")`
```

Comment:
This name collides with the appId in [the DTO](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=130920248), which has a many-to-one relationship to appName; I suspect this nomenclature is deeply entrenched in Experimenter, but since this is an interface, can we just clarify here that this isn't the same as the DTO appId?

Reply:
Wait it's not the same?  Can you provide an example?

Reply:
Yeah, the DTO doc has examples:

> Should match app_id in https://github.com/mozilla/probe-scraper/blob/master/repositories.yaml. Possible values include:
>    "org.mozilla.fenix", for Fenix nightly
>    "org.mozilla.firefox_beta", for Fenix beta
>    "org.mozilla.firefox", for Fenix release
>    "firefox-desktop", for Firefox Desktop

Reply:
Oh yep perfect ty this should not be `app_id` 👍